src/puppeteer-provider.ts

```diff
@@ -523,8 +527,12 @@ export class PuppeteerProvider {
       jobdebug(`${job.id}: Browser not needed, closing`);
       await chromeHelper.closeBrowser(browser);
 
-      jobdebug(`${job.id}: Browser cleanup complete, checking swarm.`);
-      this.checkChromeSwarm();
+      jobdebug(`${job.id}: Browser cleanup complete.`);
+
+      if (this.config.prebootChrome) {
```

Comment:
How does this work?  The prior code conditionally launched a new chrome instance if there were not enough in the swarm.  This new code seems to always launch a new instance of chrome when one closes, even when the swarm if filled to the configured concurrency amount.  Am I missing something?